cgi/import_file_process.pl

```diff
@@ -125,6 +126,8 @@
 
 my $started_t = time();
 my $import_id = $started_t;
+# sync CRM
+update_last_import_date($Org_id, $started_t);
```

Comment:
We also have nightly imports through Equadis and Agena3000, they usually call import_csv_file.pl
See for instance https://github.com/openfoodfacts/openfoodfacts-server/blob/main/scripts/imports/agena3000/run_agena3000_import.sh
The issue is that they contain multiple organizations, so it would need to be done based on the organizations actually present in the updated products.